lib/jarvis/commands/teamtime.rb

```diff
@@ -0,0 +1,12 @@
+require "clamp"
+require "jarvis/location_times"
+
+module Jarvis module Command class Teamtime < Clamp::Command
+  banner "List the current time in the team locations"
+
+  def execute
+    STDERR.puts "sdfsdf"
+    puts Jarvis::LocationTimes.new.to_a.join($/)
```

Comment:
I am not clear on whether to use `Jarvis::ThreadLogger` or regular `puts` to output regular or templated strings.

I can see that the results of the mustache template go into the GH comment and not to Hipchat output.  Is it your intention/desire to see all (multiline) HC output going via templating?